src/views/projects/Issue.svelte

```diff
@@ -1,169 +1,241 @@
 <script lang="ts">
   import type { Project } from "@app/lib/project";
-  import type { Issue } from "@app/lib/issue";
+  import type { Session } from "@app/lib/session";
+  import type { State } from "@app/lib/issue";
 
-  import Authorship from "@app/components/Authorship.svelte";
-  import Avatar from "@app/components/Comment/Avatar.svelte";
-  import Chip from "@app/components/Chip.svelte";
+  import Button from "@app/components/Button.svelte";
   import Comment from "@app/components/Comment.svelte";
-  import { formatNodeId, capitalize } from "@app/lib/utils";
-  import { formatObjectId } from "@app/lib/cobs";
+  import IssueHeader from "@app/views/projects/Issue/IssueHeader.svelte";
+  import IssueSidebar from "@app/views/projects/Issue/IssueSidebar.svelte";
+  import IssueStateButton from "@app/views/projects/Issue/IssueStateButton.svelte";
+  import Textarea from "@app/components/Textarea.svelte";
+  import { createAddRemoveArrays, Issue } from "@app/lib/issue";
+  import { isLocal } from "@app/lib/utils";
 
   export let issue: Issue;
   export let project: Project;
-</script>
+  export let session: Session | undefined;
```

Comment:
I wonder if we should not just import session directly since it's a global thing in the app?